### PR TITLE
Changes for B->D^(*)lnu code

### PR DESCRIPTION
**Pull request by dvandyk**

### CODE REVIEW

**Review by cBobeth — COMMENTED** [review]
I think we forgot to increase number of integration points in b-to-vec-l-nu.cc in line 370?! It should be from 256 -> 4096.

File: eos/b-decays/b-to-vec-l-nu.cc [hunk 1]
```diff
@@ -408,87 +408,87 @@ namespace eos
...
+        return 3.0 / 4.0 * o.vv4T();
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J4_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J4(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * -o.vv10T() / o.normalized_decay_width();
+        return 3.0 / 4.0 * -o.vv10T();
```
Comment (cBobeth) [inline eos/b-decays/b-to-vec-l-nu.cc:480]: We need to check whether there should be these minus signs in

J_4, 6s, 6c, 7, 9

Comment (dvandyk) [reply eos/b-decays/b-to-vec-l-nu.cc:480]: Check, or did you come to a consensus with @MartinJung1978 that this is not conforming to the Belle definitions?

Comment (cBobeth) [reply eos/b-decays/b-to-vec-l-nu.cc:480]: We checked that this is not conforming with the Belle 2018 definition. You might cherry-pick from my git-repo (branch BtoDstarlnu) latest commit for correction of minus signs and adaption of test cases.

Comment (cBobeth) [reply eos/b-decays/b-to-vec-l-nu.cc:480]: I try to finish another commit with a test case for B->D* lv angular observables including new physics contributions that could be also added to this pull request.

Comment (cBobeth) [reply eos/b-decays/b-to-vec-l-nu.cc]: Code changes look fine for me.

**Pull request merged**